spec/features/visiting_Chitter_test_spec.rb

```diff
@@ -0,0 +1,7 @@
+feature 'Test' do
```

Comment:
Why did you choose not to have any peeps displayed on this page for this specific feature test?